src/js/index.js

```diff
@@ -374,7 +374,15 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     },
     box: {
       responsiveBreakpoint: 'small', // when we switch rows to columns
-      // extend: undefined,
+      extend: ({ theme }) => ({
+        '::-webkit-scrollbar': {
```

Comment:
Why have we added it for boxes?

Reply:
box is div element, adding slim scroll to divs with property of overflow

Reply:
implemented new "scrollablebox" key for scrollable divs